tabular/src/autogluon/tabular/task/tabular_prediction/tabular_prediction.py

```diff
@@ -92,6 +92,9 @@ def fit(train_data,
             Note: final model returned may be fit on this tuning_data as well as train_data. Do not provide your evaluation test data here!
             In particular, when `num_bagging_folds` > 0 or `stack_ensemble_levels` > 0, models will be trained on both `tuning_data` and `train_data`.
             If `tuning_data = None`, `fit()` will automatically hold out some random validation examples from `train_data`.
+        unlabeled_data : TODO: Not sure what all the data types are that this can be.
```

Comment:
Ensure this is filled out and TODO's removed before merge

Reply:
Move this description to kwargs section, currently it is not in correct order

Reply:
Removed TODOs, moved to right spot, and put more info about arg.